Tests/SentryTests/Networking/SentryHttpTransportTests.swift

```diff
@@ -107,9 +107,13 @@ class SentryHttpTransportTests: XCTestCase {
         }
     }
 
+    class func dsn() throws -> SentryDsn {
+        try TestConstants.dsn(username: "SentryHttpTransportTests")
+    }
+
     class func buildRequest(_ envelope: SentryEnvelope) -> SentryNSURLRequest {
         let envelopeData = try! SentrySerialization.data(with: envelope)
-        return try! SentryNSURLRequest(envelopeRequestWith: SentryHttpTransportTests.dsn, andData: envelopeData)
+        return try! SentryNSURLRequest(envelopeRequestWith: dsn(), andData: envelopeData)
```

Comment:
Here's the downstream effect of removing the XCTFail from the DSN construction/retrieval in TestConstants.